src/components/sitepages/SiteData.vue

```diff
@@ -25,156 +25,236 @@
           <thumbnail-row
             class="data-thumbs"
             :images="recent_images"
+            :grouped_images="grouped_images"
             :selected_image="current_image.image_id"
             @thumbnailClicked="setActiveImage"
           />
+          <group-images-button
```

Comment:
Can we exclude this from *this* PR since we're not using it at the moment?

Reply:
Deleted!